tests/win_test.go

```diff
@@ -338,20 +357,83 @@ fmt.Scanln(&input1)
 
 	}
 
-//debug
-var input2 string
-fmt.Scanln(&input2)
-
-
 
 		})
 
 
 	})
-})
 
 
 
+Context("check visable sockets/cores/threads", func() {
+	It("testcase", func() {
+
+})})
 
 
+Context("check hyperv", func() {
+	It("testcase", func() {
+
+		var vmiHyperv tframework.VirtualMachine
+		vmiHyperv.Manifest = address_common + "/hyperv.yml"
+
+		//workaround (maybe some kind of dev env issue)
+		_, _, err := ktests.RunCommandWithNS(TestSystemNamespace, "oc", "project", TestSystemNamespace)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Create VMI using manifest with HyperV features")
+		vmiName := tframework.CreateVMorVMIfromManifest(vmiHyperv.Manifest, TestSystemNamespace)
 
+		By("Getting VMI object")
+		getVMOptions := metav1.GetOptions{}
+		vmi, err := virtClient.VirtualMachineInstance(ktests.NamespaceTestDefault).Get(vmiName, &getVMOptions)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Waiting until VMI start")
+		ktests.WaitForSuccessfulVMIStart(vmi)
+
+		By("Get vmi XML")
+		vmiXml, err := ktests.GetRunningVirtualMachineInstanceDomainXML(
+			virtClient,
+			vmi,
+		)
+
+		By("unmarshal XML")
+		domStat := &api.DomainSpec{}
+		err = xml.Unmarshal([]byte(vmiXml), domStat)
+		Expect(err).ToNot(HaveOccurred())
+			
+		By("Checking HyperV emulation features for KVM")	
+		features := domStat.Features
+               	hyperv := features.Hyperv
+                Expect((*hyperv.Relaxed).State).To(Equal("on"))
+                Expect((*hyperv.VAPIC).State).To(Equal("on"))
+                Expect((*hyperv.Spinlocks).State).To(Equal("on"))
+                Expect(*hyperv.Spinlocks.Retries).To(Equal(uint32(8191)))
+
+		By("Checking HyperV timer emulation")
+		Timer := domStat.Clock.Timer
+
+		for i, _ := range Timer {
+			fmt.Println(Timer[i].Name)
+			switch Timer[i].Name {
+			case "rtc":
+				Expect(Timer[i].TickPolicy).To(Equal("catchup"))
+				Expect(Timer[i].Present).To(Equal("yes"))
+			case "pit":
+				Expect(Timer[i].TickPolicy).To(Equal("delay"))
+				Expect(Timer[i].Present).To(Equal("yes"))
+			case "hpet":
+				Expect(Timer[i].Present).To(Equal("no"))
+
+			case "hypervclock":
```

Comment:
I think you should just check this and just ignore everything else

Reply:
I've changed it